Rubberduck.Parsing/VBA/COMReferenceManagerBase.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using NLog;
+using System.Collections.Concurrent;
+using Rubberduck.VBEditor;
+using System.Runtime.InteropServices;
+using System.IO;
+using Rubberduck.Parsing.ComReflection;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public abstract class COMReferenceManagerBase : ICOMReferenceManager
+    {
+        private const string rubberduckGUID = "{E07C841C-14B4-4890-83E9-8C80B06DD59D}";
```

Comment:
@MDoerner that's fine - the RD GUID isn't going to change on a whim anyway, and when it does, we'll have to run a full-solution search to be sure everything is A1 anyway.